adjust the recommend value of raft election-timeout in multi dc deplo… (#16561)
pingcap/docs: geo-distributed-deployment-topology.md

- To prevent remote TiKV nodes from launching unnecessary Raft elections, it is required to increase the minimum and maximum number of ticks that the remote TiKV nodes need to launch an election. The two parameters are set to `0` by default.

    ```yaml
    raftstore.raft-min-election-timeout-ticks: 50
    raftstore.raft-max-election-timeout-ticks: 60
    ```

> > **Note:**

Reviewer: ```suggestion
> **Note:** 
```

>

> > Configuring larger values for `raftstore.raft-min-election-timeout-ticks` and `raftstore.raft-max-election-timeout-ticks` can significantly decrease the likelihood of a peer on the TiKV becoming the leader. However, in a disaster scenario where some TiKV nodes are offline and the active TiKV node lags behind in Raft logs, only the Region on this specific TiKV can become the leader. In the event that the leader becomes unavailable, the Region on this TiKV node must wait for at least the duration set by `raftstore.raft-min-election-timeout-ticks' to become a new leader. It is advisable not to set these values excessively large to prevent potential impact on the cluster availability in such scenarios.

Reviewer: ```suggestion
> Using `raftstore.raft-min-election-timeout-ticks` and `raftstore.raft-max-election-timeout-ticks` to configure larger tick values for for a TiKV node can significantly decrease the likelihood of a peer on the TiKV node becoming the leader. However, in a disaster scenario where some TiKV nodes are offline and the active TiKV node lags behind in Raft logs, only the Region on this specific TiKV node with a large tick value can become the leader. In the event that the leader becomes unavailable, the Region on this TiKV node must wait for at least the duration set by `raftstore.raft-min-election-timeout-ticks' to become a new leader. It is advisable not to set these values excessively large to prevent potential impact on the cluster availability in such scenarios.
```

> > Using `raftstore.raft-min-election-timeout-ticks` and `raftstore.raft-max-election-timeout-ticks` to configure larger tick values for a TiKV node can significantly decrease the likelihood of a peer on the TiKV node becoming the leader. However, in a disaster scenario where some TiKV nodes are offline and the active TiKV node lags behind in Raft logs, only the Region on this specific TiKV node with large tick values can become the leader. In the event that the leader becomes unavailable, the Region on this TiKV node must wait for at least the duration set by `raftstore.raft-min-election-timeout-ticks' to become a new leader. It is advisable not to set these values excessively large to prevent potential impact on the cluster availability in such scenarios.

Reviewer: ```suggestion
> Using `raftstore.raft-min-election-timeout-ticks` and `raftstore.raft-max-election-timeout-ticks` to configure larger election timeout tick values for a TiKV node can significantly decrease the likelihood of a peer on the TiKV node becoming the leader. However, in a disaster scenario where some TiKV nodes are offline and the active TiKV node lags behind in Raft logs, only the Region on this specific TiKV node with large tick values can become the leader. In the event that the leader becomes unavailable, the Region on this TiKV node must wait for at least the duration set by `raftstore.raft-min-election-timeout-ticks' to become a new leader. It is advisable not to set these values excessively large to prevent potential impact on the cluster availability in such scenarios.
```

> #### PD parameters

Reviewer: ```suggestion

#### PD parameters
```

> > Using `raftstore.raft-min-election-timeout-ticks` and `raftstore.raft-max-election-timeout-ticks` to configure larger election timeout ticks for a TiKV node can significantly decrease the likelihood of a peer on the TiKV node becoming the leader. However, in a disaster scenario where some TiKV nodes are offline and the active TiKV node lags behind in Raft logs, only the Region on this specific TiKV node with large election timeout ticks can become the leader. In the event that the leader becomes unavailable, the Region on this TiKV node must wait for at least the duration set by `raftstore.raft-min-election-timeout-ticks' to become a new leader. It is advisable not to set these values excessively large to prevent potential impact on the cluster availability in such scenarios.

Reviewer 2: ```suggestion
> Using `raftstore.raft-min-election-timeout-ticks` and `raftstore.raft-max-election-timeout-ticks` to configure larger election timeout ticks for a TiKV node can significantly decrease the likelihood of Regions on that node becoming Leaders. However, in a disaster scenario where some TiKV nodes are offline and the remaining active TiKV nodes lag behind in Raft logs, only Regions on this specific TiKV node with large election timeout ticks can become Leaders. Because Regions on this TiKV node must wait for at least the duration set by `raftstore.raft-min-election-timeout-ticks' before initiating an election, it is recommended to avoid setting these values excessively large to prevent potential impact on the cluster availability in such scenarios.
```

- To prevent remote TiKV nodes from launching unnecessary Raft elections, it is required to increase the minimum and maximum number of ticks that the remote TiKV nodes need to launch an election. The two parameters are set to `0` by default.

    ```yaml
    raftstore.raft-min-election-timeout-ticks: 50
    raftstore.raft-max-election-timeout-ticks: 60
    ```

> 

Reviewer: ```suggestion
> **Note:**
>
> Configuring larger values for `raftstore.raft-min-election-timeout-ticks` and `raftstore.raft-max-election-timeout-ticks` can greatly reduce the probability that a peer on this TiKV will become the leader. However, in a disaster scenario, if some TiKV nodes are down and the active TiKV node is lagging behind in Raft logs, only the Region on this TiKV can become the leader. When the leader is down, the Region on this TiKV node needs to wait for at least the time set by `raftstore.raft-min-election-timeout-ticks' to become the leader. It is recommended that you do not set these values too large to avoid affecting the availability of the cluster in this scenario.
```

- To prevent remote TiKV nodes from launching unnecessary Raft elections, it is required to increase the minimum and maximum number of ticks that the remote TiKV nodes need to launch an election. The two parameters are set to `0` by default.

    ```yaml
    raftstore.raft-min-election-timeout-ticks: 50
    raftstore.raft-max-election-timeout-ticks: 60
    ```

> > **Note:** 

Reviewer: ```suggestion

> **Note:** 
```
